<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayReport</class>
 <widget class="QWidget" name="DayReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget{
background-color: rgb(114, 159, 207);
color: rgb(255, 255, 255);
}

#CheckTable{
font: 16px &quot;微软雅黑&quot;;
background-color: rgb(114, 159, 207);
color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>61</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit::down-button,QDateEdit::up-button{
	width:40px;
}

QWidget{
font: 18px &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>起始日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date_start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>终止日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date_finish">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="check">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>前一日</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="check_next">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>后一日</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackTable">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Page1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6">
       <item>
        <widget class="QTableWidget" name="list_table">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(114, 159, 207);
QHeadView{
	border:0px solid grey;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>物品编码</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>批号</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>物品名称</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>规格</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数量</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>单位</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>消耗类型</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>操作人</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>时间</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>物品条码</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>生产商</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>供应商</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>单价</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>合计</string>
          </property>
          <property name="background">
           <color>
            <red>114</red>
            <green>159</green>
            <blue>207</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
